src/cmake/Config.cmake.in

```diff
@@ -15,9 +15,12 @@ elseif (@OpenEXR_VERSION@ VERSION_GREATER_EQUAL 2.4 AND @FOUND_OPENEXR_WITH_CONF
     find_dependency(Threads)  # Because OpenEXR doesn't do it
 endif ()
 
-set_and_check (@PROJECT_NAME@_INCLUDE_DIR "@CMAKE_INSTALL_FULL_INCLUDEDIR@")
-set_and_check (@PROJECT_NAME@_INCLUDES    "@CMAKE_INSTALL_FULL_INCLUDEDIR@")
-set_and_check (@PROJECT_NAME@_LIB_DIR     "@CMAKE_INSTALL_FULL_LIBDIR@")
+# Compute the installation prefix relative to this file
+get_filename_component(_IMPORT_PREFIX "${CMAKE_CURRENT_LIST_DIR}/../../../" ABSOLUTE)
+
+set_and_check (@PROJECT_NAME@_INCLUDE_DIR "${_IMPORT_PREFIX}/include")
+set_and_check (@PROJECT_NAME@_INCLUDES    "${_IMPORT_PREFIX}/include")
+set_and_check (@PROJECT_NAME@_LIB_DIR     "${_IMPORT_PREFIX}/lib")
```

Comment:
Sorry, one more concern as I look this over:

In making this a relative path and specifically saying "/lib", I'm wondering what happens on systems where the library destination is different (such as lib64, or something custom set by the user with CMAKE_INSTALL_LIBDIR). Is it possible that the library destination is not actually called "lib" (or that the include destination is not called "include"), but now we've assumed and hard-coded those choices into the cmake exported config file in a way that might not reflect where the files are actually installed?

Reply:
@merwaaan Any thoughts on this? It's my only reservation about merging this PR.

Reply:
@lgritz You're right that hard-coding those directory names could create issues down the line.

I committed a change that uses the variables conveniently defined in [GNUInstallDirs](https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html).

Alternatively, are those three lines ("set_and_check") really needed?
With "modern" Cmake, we usually deal with targets instead of those old-school variables.
Other high-profile libraries that I checked do not define such variables in their config script, just targets (and it's still possible for users to get their values with CMake's `get_target_property`).
But then, maybe those variables have been here for a long time and some users depend on them.

Reply:
Yes, it's just to avoid breaking downstream projects that may have been using those variables for years. We definitely want to remove those eventually. Maybe now is a good time? But let's do it in a separate PR, because we definitely would want that breaking change to only be in master, whereas the rest of this seems safe to backport to the release branch.